packages/react-storage/src/components/StorageBrowser/Views/CreateFolderActionView/__tests__/CreateFolderActionView.spec.tsx

```diff
@@ -46,7 +46,7 @@ describe('CreateFolderActionView', () => {
 
     waitFor(async () => {
       const input = screen.getByLabelText('Enter folder name:');
-      const button = screen.getByRole('button', { name: 'Create folder' });
+      const button = screen.getByRole('button', { name: 'Create Folder' });
```

Comment:
should this be `Set Destination`?

```suggestion
      const button = screen.getByRole('button', { name: 'Set Destination' });
```

Reply:
Just made the changes that were necessary to get the existing tests to pass and don't want to touch them again in this PR. FWIW the "destination" language probably need to be reworked since the destination is the prefix